lib/CL/devices/common.c

```diff
@@ -125,6 +126,19 @@ llvm_codegen (char *output, unsigned device_i, cl_kernel kernel,
   pocl_cache_final_binary_path (final_binary_path, program, device_i, kernel,
                                 command, specialize);
 
+#ifdef _WIN32
+  // Avoid race condition of multiple processes accessing same files.
+  char MtxName[MAX_PATH];
```

Comment:
We have POCL_MAX_PATHNAME_LENGTH, is it the same (or at least as large as) MAX_PATH?

Reply:
MAX_PATH is 260.